install/kubernetes/README.md

```diff
@@ -1,48 +1,9 @@
 # Deployment of Hubble alongside Cilium
 
-This `hubble.yaml` contains a Kubernetes DaemonSet which will deploy Hubble as
-a pod in the `kube-system` namespace on each node. It assumes that each node
-is running Cilium with it's monitor socket available under the host path
-`/var/run/cilium/`. This holds true if Cilium is deployed via
-its standard YAML manifests.
+Starting with **Cilium >=1.8**, the Hubble Helm charts are now part of Cilium.
+Please refer to the Cilium [installation instructions](https://docs.cilium.io/en/latest/gettingstarted/#installation)
+for details.
 
-## Standard Deployment
-
-To deploy the Hubble server DaemonSet, use the following
-[Helm](https://helm.sh/) command:
-
-    helm template hubble \
-        --namespace kube-system \
-        > hubble.yaml
-    kubectl apply -f hubble.yaml
-
-Set `--listen-client-urls` option to start the Hubble server on a TCP port
-instead of unix domain socket. For example, to listen to port 8080 on localhost:
-
-    --set listenClientUrls='{localhost:8080}'
-
-## Metrics
-
-1. Metrics can be enabled via the `metrics.enabled` helm option:
-
-        helm template hubble \
-            --namespace kube-system \
-            --set metrics.enabled="{dns:query,drop,tcp,flow,port-distribution}" \
-            > hubble.yaml
-        kubectl apply -f hubble.yaml
-
-2. Deploy Prometheus & Grafana
-
-   The following deploys Prometheus and Grafana into the `cilium-monitoring`
-   namespace:
-
-       kubectl apply -f https://raw.githubusercontent.com/cilium/cilium/v1.6/examples/kubernetes/addons/prometheus/monitoring-example.yaml
-
-   Import the dashboard (`install/kubernetes/grafana.json`) via *Create* ->
-   *Import*
-
-## Usage
-
-To query Hubble with the CLI on the first node:
-
-    kubectl exec -n kube-system -t -c hubble ds/hubble -- hubble observe --since 1s
+> If you are running Cilium 1.7, you will find the Helm charts and
+> installation instructions on how to deploy Hubble in [the v0.5
+> branch of this repository](https://github.com/cilium/hubble/tree/v0.5/install/kubernetes).
```

Comment:
Why is this paragraph quoted?

Reply:
To visually separate it from the above paragraph. But I can see that this is semantically confusing, fixed.